src/middleware/error-handler/index.js

```diff
@@ -5,10 +5,11 @@ export const DEFAULT_ERROR_MESSAGE = 'Something went wrong';
 // eslint-disable-next-line no-unused-vars
 export default function errorHandler(err, req, res, next) {
     const { logger } = getContainer();
-    const environment = getConfig('logger.loggly.environment');
-    const displayErrorDetails = !['prod', 'demo'].includes(environment);
+    const hideErrorDetails = getConfig('logger.hideErrorDetails');
 
-    if (process.env.NODE_ENV !== 'test') {
+    const displayErrorDetails = !hideErrorDetails;
+
+    if (!displayErrorDetails) {
```

Comment:
@rh-glob you should leave this check as it was. It is used to prevent putting too much noise on the terminal when running jest tests. This is not related with TEST environment but with NODE_ENV local environment variable that it is set automatically to this value when jest is running.